warehouse/upload.go

```diff
@@ -1194,9 +1196,49 @@ func (job *UploadJobT) setUploadColumns(opts UploadColumnsOpts) (err error) {
 	return err
 }
 
+func (job *UploadJobT) triggerUploadNow() (err error) {
+	job.uploadLock.Lock()
+	defer job.uploadLock.Unlock()
+	upload := job.upload
+	//TODO: can it be in waiting state only?
+	newjobState := Waiting
+	var metadata map[string]string
+	unmarshallErr := json.Unmarshal(upload.Metadata, &metadata)
+	if unmarshallErr != nil {
+		metadata = make(map[string]string)
+	}
+	metadata["nextRetryTime"] = time.Now().Add(-time.Hour * 1).Format(time.RFC3339)
+	metadataJSON, err := json.Marshal(metadata)
+	if err != nil {
+		return err
+		//metadataJSON = []byte("{}")
+	}
+
+	uploadColumns := []UploadColumnT{
+		{Column: "status", Value: newjobState},
+		{Column: "metadata", Value: metadataJSON},
+		{Column: "updated_at", Value: timeutil.Now()},
+	}
+
+	txn, err := job.dbHandle.Begin()
+	if err != nil {
+		panic(err)
+	}
+	err = job.setUploadColumns(UploadColumnsOpts{Fields: uploadColumns, Txn: txn})
```

Comment:
can we use `job.setUploadStatus` instead?

Reply:
Since other columns are being updated alongside status. preferred this way...